mountpoint-s3-client/src/s3_crt_client.rs

```diff
@@ -130,13 +132,16 @@ impl S3CrtClient {
             Endpoint::from_region(region, AddressingStyle::Automatic)?
         };
 
+        let request_payer = config.request_payer;
+
         Ok(Self {
             allocator,
             s3_client,
             event_loop_group,
             endpoint,
             next_request_counter: AtomicU64::new(0),
             user_agent_header,
+            request_payer
         })
```

Comment:
Prefer to inline this, and also to maintain trailing comma.

```suggestion
        Ok(Self {
            allocator,
            s3_client,
            event_loop_group,
            endpoint,
            next_request_counter: AtomicU64::new(0),
            user_agent_header,
            request_payer: config.request_payer,
        })
```